src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -52,5 +52,4 @@ public CommandResult execute() {
         }
         return matchedPersons;
     }
-
```

Comment:
Avoid auto-formatting unrelated parts of the code. Doing so can result in unrelated changes being included in the commit. You can use sourcetree to look through changes before committing, and discard any unrelated changes. If you wish to commit formatting changes to unrelated parts of the code, do so in a separate commit.